src/components/ReportActionItem/MoneyRequestView.js

```diff
@@ -228,6 +229,43 @@ function MoneyRequestView({report, parentReport, parentReportActions, policyCate
     const pendingAction = lodashGet(transaction, 'pendingAction');
     const getPendingFieldAction = (fieldPath) => lodashGet(transaction, fieldPath) || pendingAction;
 
+    const getErrorForField = useCallback(
+        (field) => {
+            // Checks applied when creating a new money request
+            // NOTE: receipt field can return multiple violations, so we need to handle it separately
+            const fieldChecks = {
+                amount: {
+                    isError: transactionAmount === 0,
+                    translationPath: 'common.error.enterAmount',
+                },
+                merchant: {
+                    isError: isPolicyExpenseChat && isEmptyMerchant,
+                    translationPath: 'common.error.enterMerchant',
+                },
+                date: {
+                    isError: transactionDate === '',
+                    translationPath: 'common.error.enterDate',
+                },
+            };
+
+            const {isError, translationPath} = fieldChecks[field] ? fieldChecks[field] : {};
```

Comment:
```ts
fieldChecks[field] || {};
```

I think that silencing the linter is better than working it around.

Reply:
done!